spec/controllers/export_files_controller_spec.rb

```diff
@@ -0,0 +1,19 @@
+RSpec.describe ExportFilesController, type: :controller do
+
+  let(:user) { FactoryGirl.create(:user) }
+  let(:component) { FactoryGirl.build(:component) }
+
+  before do
+    component.roles.grant type: "Viewer", agent: "public"
+    component.save!
+    sign_in user
+  end
+
+  describe "success" do
+    it "sends email to the user" do
```

Comment:
Is this the right description for this test?  It seems to be testing an HTTP response, not whether an email was sent.

Reply:
Oh yeah